### ISSUE TITLE
Versions after 1.1.3 break 4cs sound player

### ISSUE THREAD

**Issue by BallM4788** [zixaphir/appchan-x#218]
Appchan X 1.1.3 initial release (https://github.com/zixaphir/appchan-x/blob/7f974593f5/appchan_x.user.js) with the latest build of 4cs (https://github.com/dnsev/4cs/blob/master/web/4cs.user.js)
![ 0 _q_-_pol_is just ridiculous_20130326-184549](https://f.cloud.github.com/assets/3979617/305887/f6fc99d2-9666-11e2-97cf-f56cd0d4a46b.png)

The same day, the next update completely broke appchan x as I'm sure you are aware:
https://github.com/zixaphir/appchan-x/blob/177e487bf2/appchan_x.user.js with the same version of 4cs as above
![ 0 _q_-_pol_is just ridiculous_20130326-185916](https://f.cloud.github.com/assets/3979617/305932/dce258f0-9668-11e2-88e2-23322cf37fcd.png)

Fast forward to Appchan X 1.2.5. Almost everything is back to normal, except for 4cs's menu which is sitting smack dab at the bottom of the page:
https://github.com/zixaphir/appchan-x/blob/master/appchan_x.user.js with, again, the same version of 4cs as in the images above
![ 0 _q_-_pol_is just ridiculous_20130326-184327](https://f.cloud.github.com/assets/3979617/305876/a5bb9ff0-9666-11e2-8806-3f45a9f1005b.png)

In all three of these screencaps, Appchan X and 4cs are the only two userscripts running. Because the same version of 4cs is used in all three examples, and settings and preferences for both scripts remaining constant throughout, I feel it's reasonable to conclude that it must be something in Appchan X that's causing the issue.

TL;DR FOUND BUG, PLZ FIX :3c
